source/Node.js

```diff
@@ -91,7 +91,9 @@ function getPath ( node ) {
         if ( id = node.id ) {
             path += '#' + id;
         }
-        if ( className = node.className.trim() ) {
+
+
+        if (node.className && node.classList.trim && (className = node.className.trim()) ) {
```

Comment:
This doesn't look right. 
1) you're checking for trim on classList, and then using it on className
2) assignments inside if conditions are disgusting